go-get.go

```diff
@@ -0,0 +1,6 @@
+// This file is here so go get succeeds, as without it errors with:
+// no buildable Go source files in ...
+//
+// +build !windows
+
+package wmi
```

Comment:
Well, it won't `go get` any dependencies of *this* project (obviously, since there was nothing to compile), but it will continue with other dependencies of other packages.